gas_cost.txt

```diff
@@ -20,4 +20,27 @@
 |  HappyRedPacket_ERC721                        ·          -  ·          -  ·    1588214  ·      19.9 %  ·          -  │
 ················································|·············|·············|·············|··············|··············
```

Comment:
I think it would be better if we can keep the test result in a single file, like:
[/test/performance_test.txt](https://github.com/DimensionDev/RedPacket/blob/97512dc380c675d1230a50fa1dd3b8164eb5a0e7/test/performance_test.txt), for these reasons:
1. Keep it in a separate directory, keeping source files organized
2. Keep test results in a single file, easier for comparison/`reviewing its changing history`, etc